lib/blockchain/cardano-build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+if [ -z "$USER" ]; then
+    USER=`whoami`
+fi
+
+tmpdir=$(mktemp -d -t nix-XXXXXXXXXX)
+cd $tmpdir
+wget -q https://nixos.org/nix/install
```

Comment:
They way the script is currently written, if any command fails, it will keep going with the next one. I think it shouldn't do that ...